### PR TITLE
Add Bump to the Addon page

### PR DESCRIPTION
**Pull request by haiman233**

### CODE REVIEW

File: pages/Addons.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ SfChunkInfo | FN_FAL113 | <details><summary>expand</summary>Scan the amount of s
...
 Missile Warfare | Kaiyan2006 | <details><summary>expand</summary>Missile Warfare adds a bunch of missiles to Slimefun that can be fired at others!</details> | N/A | [GitHub](https://github.com/koiboi-dev/MissileWarfare) | [Releases](https://github.com/koiboi-dev/MissileWarfare/releases)
 SlimefunAdvancements | char321 | <details><summary>expand</summary>SlimefunAdvancements adds fully configurable advancements to Slimefun. Easily create your own progression!</details> | [Readme](https://github.com/qwertyuioplkjhgfd/SlimefunAdvancements/blob/main/README.md) | [Github](https://github.com/qwertyuioplkjhgfd/SlimefunAdvancements) | [Builds](https://thebusybiscuit.github.io/builds/qwertyuioplkjhgfd/SlimefunAdvancements/main/)
+Bump | SlimefunGuguProject | <details><summary>expand</summary>A Slimefun4 addon that adds magic weapon, food and appraisal system.Through the appraisal system, you could even get the 8 stars weapon or armor.</details> | [Readme](https://github.com/SlimefunGuguProject/Bump/blob/main/README.md) | [Github](https://github.com/SlimefunGuguProject/Bump) | [Builds](https://builds.guizhanss.net/SlimefunGuguProject/Bump/main/)
```
Comment (Sefiraat) [inline pages/Addons.md:75]:
```suggestion
Bump | SlimefunGuguProject | <details><summary>expand</summary>A Slimefun4 addon that adds magic weapons, food and an appraisal system. Through the appraisal system, you could even get an 8 star weapon or armor.</details> | [Readme](https://github.com/SlimefunGuguProject/Bump/blob/main/README.md) | [Github](https://github.com/SlimefunGuguProject/Bump) | [Builds](https://builds.guizhanss.net/SlimefunGuguProject/Bump/main/)
```

**Comment by haiman233** [comment]
thx for a lot

**Review by Phoenix-Starlight — APPROVED** [review]

**Review by Phoenix-Starlight — APPROVED** [review]
Conflict resolved.

**Comment by svr333** [comment]
is it intended that the builds page is not in english?

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]
"Builds" links to a third-party website here. This could be misleading as people may think it would link to OUR builds page.

I am not sure if we want to link to sites other than GitHub, CurseForge, Spigot or our own builds page in the first place. So I am rejecting this for now.
If you were to publish your addon on Spigot, CF or GitHub, feel free to make a new PR.

**Comment by ybw0014** [comment]
Hi, I wonder if you would accept changing the `Builds` to `External Builds` or `Builds (External)`.
The external builds page is modified from your builds page and is fully open source (https://github.com/ybw0014/maven-builds). Currently, it has no other language than Simplified Chinese. I may add the i18n module to it asap.

**Comment by Phoenix-Starlight** [comment]
I would accept that change, as it would notify users that it links to a third-party.

**Pull request closed without merging**